sdk/iOS/src/MSPullSettings.m

```diff
@@ -0,0 +1,37 @@
+// ----------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// ----------------------------------------------------------------------------
+
+#import "MSPullSettings.h"
+
+#pragma mark * MSPullSettings Implementation
+
+@implementation MSPullSettings
+
+static NSInteger const _defaultPageSize = 50;
+
+#pragma mark * Initializer method(s)
+
+- (id)init {
+    self = [super init];
+    if (self) {
```

Comment:
Changed the return type to instancetype. 
Not moving initialization to the getter as it is not really a perf impacting operation. In such a case, I prefer initializing (especially with constant values) in the initializer.